src/game/common/rts/player.cpp

```diff
@@ -100,25 +120,476 @@ Player::~Player()
 // zh: 0x00457C80, wb: 0x00861D13
```

Comment:
In constructor, change `m_upgradeList(0)` to `m_upgradeList(nullptr)`

Reply:
fixed.